packages/components/__mocks__/uuid.ts

```diff
@@ -11,9 +6,5 @@ function fakeUUID() {
   return `123e4567-e89b-12d3-a456-${finalSection + count}`;
 }
 
-const uuid = fakeUUID as UUIDFunction;
-
-uuid.v1 = fakeUUID;
-
-// eslint-disable-next-line import/no-default-export
-export default uuid;
+export const v1 = fakeUUID;
+export const v4 = fakeUUID;
```

Comment:
Question for my understanding:  How is `v4` being used here?  Throughout all the changes it looks like we needed to use only `v1`

Reply:
Did not dig into it. If you have it running, try removing it and run the test.

Reply:
Tests still pass!

Reply:
huh, i guess it doesn't need to be there then

Reply:
Seems that tests also break when you specifically use v4 instead of v1

Reply:
that's a problem for when you actually need to use v4 or v3 or other v's really.